packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/expiring-contract-rolls/Roll_1_Mining_Rewards.json

```diff
@@ -0,0 +1,280 @@
+{
```

Comment:
Sorry, can you explain what this file is relative to the other file?

Reply:
This file contains the roll output. the shareholder payout here is the payouts when the considering only the period over the two pools (the roll). This is why it's in the expiring contracts roll directory